nbs/methods.ipynb

```diff
@@ -13,7 +13,7 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.    
</pre><p>We should also add a <code>Returns</code> section describing the output of each method. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I agree, I added some simple description for the Returns section, feel free to change